test/unit/org/apache/cassandra/db/DirectoriesTest.java

```diff
@@ -151,15 +151,15 @@ public void beforeTest() throws IOException
     @AfterClass
     public static void afterClass()
     {
-        FileUtils.deleteRecursive(tempDataDir);
```

Comment:
opportunistic fix as the used method was marked as deprecated, similar cases in this PR at various places